easybuild/easyconfigs/p/pypmt/pypmt-1.1.0-foss-2023a.eb

```
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'pypmt'
+version = '1.1.0'
+
+homepage = 'https://git.astron.nl/RD/pmt'
+description = """PMT is a high-level software library capable of 
+    collecting power consumption measurements on various hardware."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Since this EasyConfig does not require MPI, I think we could move this in the module hierarchy to the `gfbf` toolchain (requires also a rename the file to `pypmt-1.1.0-gfbf-2023a.eb`):

```suggestion
toolchain = {'name': 'gfbf', 'version': '2023a'}
```